src/db/migrations/20181126141320-create-user.js

```diff
@@ -40,6 +40,14 @@ export default {
       type: Sequelize.STRING,
       allowNull: true
     },
+    passwordExpiresIn: {
```

Comment:
Why do you have passwordExpiresIn and token here? These colums are no longer needed.

Reply:
@okoroemeka, nice observation. I've taken them out